### PR TITLE
Erekir units rebalanced for pvp

### PR DESCRIPTION
**Pull request by nanmenyangde**
Rebalanced some units in order to fit for recent unit changes for pvp

If your pull request is **not** translation or serverlist-related, read the list of requirements below and check each box:

- I have read the [contribution guidelines](https://github.com/Anuken/Mindustry/blob/master/CONTRIBUTING.md).
- I have ensured that my code compiles, if applicable.
- I have ensured that any new features in this PR function correctly in-game, if applicable.

### CODE REVIEW

File: core/src/mindustry/content/UnitTypes.java [hunk 1]
```diff
@@ -3767,7 +3767,7 @@ public static void load(){
             speed = 1.1f;
             rotateSpeed = 3.2f;
             accel = 0.1f;
-            health = 8000f;
+            health = 5500f;
```
Comment (Darkness6030) [inline core/src/mindustry/content/UnitTypes.java:3770]: -3500 HP from quell? Are you sure?
btw i agree with all other changes, but this...

**Review by Darkness6030 — CHANGES_REQUESTED** [review]
Maybe 8000 -> 7200? Totally not 5500 👀

**Comment by Anuken** [comment]
This needs review from other players, and rationale for every change.

File: core/src/mindustry/content/UnitTypes.java (hunk 1 above)
Comment (nanmenyangde) [reply core/src/mindustry/content/UnitTypes.java:3770]: i think beryllium walls are too cheap for such a high health so i decreased it greatly.i think 7200 is also ok if we build quells with tungsten walls

**Comment by Cry0flu1d** [comment]
but tbh dude, the tank units need to buff

**Comment by nanmenyangde** [comment]
> but tbh dude, the tank units need to buff

t3 tanks need buff while t2 tanks need debuff. As for other tank units i'm not sure

by the way, since the bug that collaris can damage air units has been fixed, i decide to undo my modification on its speed

**Comment by Cry0flu1d** [comment]
> > but tbh dude, the tank units need to buff
> 
> t3 tanks need buff while t2 tanks need debuff. As for other tank units i'm not sure
> 
> by the way, since the bug that collaris can damage air units has been fixed, i decide to undo my modification on its speed

anyway plz buff vanquish , it has considerably low damage output compares to tecta (mainly because of its long cooldown time of the main cannon) , and it seldom appears in PvPs except some special circumstances

**Comment by Larofeticus** [comment]
Meh. These changes don't align with any of the balance problems I'm seeing on my server.

Stells don't have a unique role. (My personal idea for that is bullets hitting tanks should deplete an extra pierce. Nice defensive/tip of the spear role to that.)

Elude rushes are devastating and the only defense is to wall off completely.

Locus is overpowered relative to T2s and even T3s.

But the main thing that needs fixed is the relative cost and effectiveness of units vs. turrets.

There are units, the best turrets are units, and then there are buildings that shoot lights for cosmetic purposes, are very expensive and slow to build, and don't affect units in a meaningful way.

**Comment by Anuken** [comment]
More discussion on balancing is good, but this PR will not be merged either way - I'm not going to be nerfing or buffing units based on the opinions of one person alone.

**Comment by Larofeticus** [comment]
Could you share some "intent" for unit/turret roles so that people could tune their suggestions to be more in line with what you're trying to accomplish?

I'm thinking things like "The role of unit X is _sentence_. It should be good at task W and X, but bad at tasks Y and Z." The same could go for the different turrets; some of them I can't figure out what it's supposed to be good at.

**Comment by Anuken** [comment]
Haha... the thing is, there was no intent to begin with. Anything goes. I can't provide any further guidance besides the obvious ("it's a tank so it should do tank stuff like have a lot of armor and be slow").

**Comment by IndeedIExist** [comment]
> Haha... the thing is, there was no intent to begin with. Anything goes. I can't provide any further guidance besides the obvious ("it's a tank so it should do tank stuff like have a lot of armor and be slow").

To me is a massive red flag, make a flow chart, following a fictional theme is a massive detriment to game balance, think of theme to be the result of its proper use.(look to rl mech infantry for proof) additionally i think that the major draw for players in v5 and less in v6 was the pvp(not due to weak campaign(custom maps))  . i have dedicated a--lot(but not exclusively) of time to this aspect of your game. it may not show in in the campaign but it can and will be exploited in a competitive scenario, these by the way are only the result of players enjoying your game enough to dedicate large chunks of time to prepare and, well compete. it would be a shame for me to see a game i have well over a thousand hours in to looses the charm that initially got me dedicated. take your time and dont ignore us as a competitive community

**Comment by nanmenyangde** [comment]
> > > but tbh dude, the tank units need to buff
> > 
> > 
> > t3 tanks need buff while t2 tanks need debuff. As for other tank units i'm not sure
> > by the way, since the bug that collaris can damage air units has been fixed, i decide to undo my modification on its speed
> 
> anyway plz buff vanquish , it has considerably low damage output compares to tecta (mainly because of its long cooldown time of the main cannon) , and it seldom appears in PvPs except some special circumstances

but don't forget - tecta seldoms appears in PvPs either. all t4 units are not powerful in 139 both for the collaris' great power and their weakness facing a number of lucuses, anthicuses and obviates. quells' appearance is an exception just due to the low price of beryllium, not its strength

**Comment by Darkness6030** [comment]
@nanmenyangde @Larofeticus and others

Since units are *entities* and are created only server-side, you can just change their parameters in a plugin. If *you* think that "Unit1" is OP and "Unit2" is bad, just buff or debuff them on your server.

**Comment by Anuken** [comment]
> make a flow chart

Of what? That is *incredibly* vague.

> following a fictional theme is a massive detriment to game balance

What are you talking about? What fictional theme? I literally said, "anything goes".

**Comment by Larofeticus** [comment]
In practice, "anything goes" is actually "nothing goes"

**Comment by Anuken** [comment]
Fantastic statement, except I'm not sure what that has to do with what I was replying to, or your question in general. You asked what my intent was, and *I answered* - you are free to suggest assigning any *intent* to units that you find fitting.

**Comment by Anuken** [comment]
> Making units just for a sake of making units is quite inconsiderate for balance.

That statement is more or less irrelevant now, as the units are already made.

> because we are not maintainers

And how does that stop you from having opinions or suggestions, exactly?

You say "maintainers", but there is really only one person who *maintains* things - me - and I'm not exactly thrilled at the prospect of spending my limited time and motivation on making flowcharts to satisfy a rather small portion of the community, when there are a million other (obvious, more pressing, more interesting) problems to solve. This is not something that's going to happen at this stage of development (*right before a release*) or without community help.

> and especially when you're "not going to be nerfing or buffing units based on the opinions of one person alone."

I don't quite understand what your problem with my statement is here. Are you saying I should simply take the first balancing suggestion that comes up without *checking to see what other people think about it?* Are you saying I should have immediately merged this PR, despite the fact that multiple people have showed up to say "meh, these changes don't align with what I want"? Because *if not*, then what's the problem with *considering multiple people's opinions first*?

**Comment by LeoSko** [comment]
> the units are already made

> at this stage of development (right before a release)

Is this the first version of the game or am I missing something?

> I'm not exactly thrilled at the prospect of spending my limited time and motivation on making flowcharts to satisfy a rather small portion of the community

Just add to the wiki a strong and weak points for every unit/building. Like `mace is strong against: pulsars, scorch; weak against: horizon`.

> I don't quite understand what your problem with my statement is here.

Please, read the article. :) Basically community can't decide, because we are a lot of people and we will always have different opinions.

**Comment by Anuken** [comment]
> Is this the first version of the game or am I missing something?

What does that have to do with what I said?

> Just add to the wiki a strong and weak points for every unit/building. 

"Just"? Every notable combination of unit and building in the game..? Not only would it be inaccurate at first, but it would require significant iteration and testing just to get "correct" - and that's just to get a *list*, not even *start balancing*. I am not interested in such an undertaking.

> Please, read the article. :) Basically community can't decide, because we are a lot of people and we will always have different opinions.

I didn't get that impression. Something like that was *mentioned*, but not in a "the community can't decide anything" kind of way - both of these apply to very specific cases.

image

image


---

Anyway - I'm not interested in debating this further. If you want balance, make a suggestion, explain your reasoning. That is all. Bye.

**Pull request closed without merging**